src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -83,7 +88,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
         }
 
         private CSharpParseOptions(CSharpParseOptions other) : this(
-            languageVersion: other.LanguageVersion,
+            languageVersion: other.SpecifiedLanguageVersion,
```

Comment:
There is no constructor that avoids recomputing it, so that would have to be added to take this suggestion. That would be an unjustified micro-optimization of code that isn't on a hot path, so it is exactly the kind of thing we should avoid. [http://c2.com/cgi/wiki?PrematureOptimization](http://c2.com/cgi/wiki?PrematureOptimization) 

---
In reply to: [74499611](https://github.com/dotnet/roslyn/pull/12969#discussion_r74499611) [](ancestors = 74499611,74492130)